Sources/FoundationEssentials/JSON/BufferView.swift

```diff
@@ -100,7 +102,8 @@ extension BufferView:
     RandomAccessCollection {
 
     typealias Element = Element
-    typealias Index = BufferViewIndex<Element>
+    // Index is already defined
+    // typealias Index = BufferViewIndex<Element>
```

Comment:
(nit) we can probably remove these two comments.

Reply:
@iCharlesHu Thank you. The comments is removed.
https://github.com/apple/swift-foundation/pull/96/commits/78161e185ca76e5c48953d5a27f36472a99eba05